### PR TITLE
Turn status indicator

### LINKED ISSUE #33: Spin up bootstrap/stylesheet
Get bootstrap up and running and setup a theme.

### LINKED ISSUE #34: Turn Status Indicator
Implement the turn status indicator on the front end

### PR DESCRIPTION
**Pull request by gassajor000**

### CODE REVIEW

**Comment by gassajor000** [comment]
closes #34, closes #33

File: trouble_client/src/models/Player.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+
+/** Represents a player in the game. */
+class Player{
```
Comment (simsinght) [inline trouble_client/src/models/Player.js:3]: is this how we'll track if we are current player?

File: trouble_client/src/game/socket_context/index.js [hunk 2]
```diff
@@ -2,14 +2,15 @@ import React, { useState, useEffect } from "react";
...
+import SpaceState from "../../models/SpaceState";
 import { PlayerColors } from "../../Colors";
+import Player from "../../models/Player";
 
 const SocketProvider = (props) => {
     const [value, setValue] = useState({
       rollResult: 0,
-      currentPlayer: "Player 1",
-      completedPlayer: "Player 2",
+      currentPlayer: new Player("Brad Pitt", PlayerColors.blue),
```
Comment (simsinght) [inline trouble_client/src/game/socket_context/index.js:12]: so we would compare if currentPlayer.name == my.name of some kind? just trying to figure out how we know when it's "my" turn.

**Review by simsinght — APPROVED** [review]

File: trouble_client/src/models/Player.js (hunk 1 above)
Comment (gassajor000) [reply trouble_client/src/models/Player.js:3]: Partially. This is just a class for representing a player on the front end.

File: trouble_client/src/game/socket_context/index.js (hunk 2 above)
Comment (gassajor000) [reply trouble_client/src/game/socket_context/index.js:12]: Yeah something like that. The other alternative is the server explicitly tells you it's your turn.

**Pull request merged**